dev-games/godot/godot-2.1.6.recipe

```diff
@@ -7,11 +7,11 @@ click to a number of platforms, including the major desktop platforms \
 (Linux, Mac OSX, Windows) as well as mobile (Android, iOS) and web-based \
 (HTML5) platforms."
 HOMEPAGE="https://godotengine.org/"
-COPYRIGHT="2007-2017 Juan Linietsky, Ariel Manzur"
+COPYRIGHT="2007-2019 Juan Linietsky, Ariel Manzur"
```

Comment:
Check your screenshot

Reply:
Yes, copyright is correct and I'm using godot 2.1.6,. Screenshots were done with 2.1.5 on same machine prior until I replaced it with the 2.1.6 build.

Reply:
Fixed copyright